src/components/NewTransaction.js

```diff
@@ -3,6 +3,14 @@ import React from "react";
 const NewTransaction = () => {
   return <div>
     <h2>New transaction</h2>
+    <label for="title">Title</label>
```

Comment:
maybe this should change to "Label" instead of title
as the label of the transaction, what do you think

Reply:
I think "Title" suits better here.

Reply:
alright